.github/workflows/ci-pr.yaml

```diff
@@ -96,7 +99,7 @@ jobs:
           popd
         done
   deployment-tests:
-    runs-on: [self-hosted, live]
+    runs-on: [self-hosted, gh-actions-ci]
```

Comment:
Change the host label to something like `live-v2`, so post merge no need to change anything